src/compile/selection/transforms/project.ts

```diff
@@ -56,6 +55,31 @@ const project: TransformCompiler = {
       return {[range]: signals[sg] = sg};
     };
 
+    // If no explicit encodings are specified, set some defaults.
```

Comment:
if not explicit "projection (either fields or encoding)"?

Reply:
Fixed.